<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome </string>
   </property>
  </widget>
  <widget class="QLabel" name="totalItemsValue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Items = </string>
   </property>
  </widget>
  <widget class="QLabel" name="lowStockAlertsValue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Stock Items = </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
